src/assets/scss/mixins.scss

```diff
@@ -0,0 +1,9 @@
+@mixin backgroundImg($url) {
+  background: url($url) 100% 100% no-repeat;
+  background-size: cover;
+}
+
+@mixin size($width,$height: $width) {
```

Comment:
Mixins are also better be used if you define them (there are a lot of cases when you could use the mixin, but set width and height without it)